File: CampChallenge_PHP/009.関数/03_argument_2.php
<?php
	function calc($a, $b=5, $flag=false) {
		$result = $a * $b;
		if ($flag) {
			$result *= $result;
		}

		echo $result . "<br>\n";
	}

	calc(10, 2, true);
	calc(10, 2);
	calc(10);

File: .old_20190514/カリキュラム/CampChallenge_Ruby/009.クラス/03_argument2.rb
class Calc
	def multi(a, b=5, flag=false)
		ans = a * b
		ans *= ans if flag
		return ans
	end
end

calc = Calc.new

puts calc.multi(2, 3, true)
puts calc.multi(2, 3, false)
puts calc.multi(2, 3)
puts calc.multi(2)
